cloudbuild-k8s-generator.py

```diff
@@ -183,7 +183,7 @@ def main():
 
   # TODO(PR/346): Spark operator: "make app/verify" fails
   # TODO(PR/516): SonarQube is flaky
```

Comment:
If you are fixing this issue, remove it from TODO

Reply:
Done